### ISSUE TITLE
Cannot find module 'rxjs-compat/add/operator/merge'

### ISSUE THREAD

**Issue by phsims** [ngrx/platform#1073]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x ] Bug report  
[ ] Feature request
[ ] Documentation issue or request
</code></pre>

## What is the current behavior?
I'm not able to generate the store with angular 6 and ngrx.
`ng generate store State --root --module app.module.ts --collection @ngrx/schematics`
produces the following error
```
Cannot find module 'rxjs-compat/add/operator/merge'
Error: Cannot find module 'rxjs-compat/add/operator/merge'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\halliganp\Documents\GitHub\FusionII\node_modules\rxjs\add\operator\merge.js:3:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)

```
## Expected behavior:
expect store files to be geneerated

## Minimal reproduction of the problem with instructions:
fresh installs of angular, node, npm updated angular CLI app generated.

**Comment by tdeschryver** [comment]
The NgRx 'Angular 6 version' is currently in beta (you can install it via `npm i @ngrx/xxx@next` or via the angular cli with `ng update @ngrx/store --next`.
If you don't want the beta version you can install `rxjs-compat`.

**Comment by phsims** [comment]
`ng update @ngrx/store --next` doesnt seem to be doing anything so I tried `rxjs-compat` but get the following error

```
Schematic input does not validate against the Schema: {"dryRun":false,"root":true,"module":"app.module.ts","collection":"@ngrx/schematics","path":"app","sourceDir":"src","flat":true,"spec":true,"statePath":"reducers"}
Errors:

  Data path "" should have required property 'name'.
```

**Comment by tdeschryver** [comment]
The name issue is an issue that is resolved in the beta version - see #1033.
